default.py

```diff
@@ -207,7 +215,7 @@ def PLAY(payload, params):
     })
 
     __set_last_watched(anime_url, is_dubbed, name, image)
-    control.play_source(s.get_video_link(), watchlist_update(episode, kitsu_id), on_stopped)
+    control.play_source(s.get_video_link(), watchlist_update(episode, kitsu_id), on_stopped, on_percent)
```

Comment:
you can pass here None if not control.getSetting('watchlist.percentbool'),
then callback can just `return int(control.getSetting('watchlist.percent'))`